docs/fastdds/dds_layer/core/policy/policy.rst

```diff
@@ -10,6 +13,10 @@ independently.
 However, there may be cases where several policies conflict. Those conflicts are notified to the user through the
 `ReturnCodes` that the QoS setter functions returns.
```

Comment:
Unrelated to the change. Cannot find `ReturnCode_t` in the API reference.

Reply:
Because its on `eprosima::fastrtps::types`

Reply:
:thinking: That sucks!